COMMANDS.md

```diff
@@ -29,6 +29,7 @@ Command Syntax | Plugin class | Description | Working?
 `!cbsg` | CBSG | Spew some corporate bullshit from the [Corporate Bullshit Generator](http://cbsg.sourceforge.net/cgi-bin/live). | Yes
 `!channel <commands>` | ChannelAdmin | Administrate the current channel. [See below](#chanadmin). | Yes
 `!clickbait` | Clickbait | Generate a clickbait-y title. | Yes
+`!cstopic <cs|system|crypto|random>` | Topic | Generate worthy CS,System,Crypto Research Topics. | Yes
```

Comment:
This should probably just stay as `!topic`, because `!cstopic cs` sounds strange.